src/mmw/apps/modeling/tasks.py

```diff
@@ -56,7 +56,7 @@ def start_histograms_job(polygons):
     }
 
 
-@shared_task(bind=True, default_retry_delay=1, max_retries=20)
+@shared_task(bind=True, default_retry_delay=1, max_retries=42)
```

Comment:
There is a slight delay between when the clock starts in the front-end and when it does in the back-end.  The attempt here is to have them both finish at roughly the same time, with the back-end finishing slightly before.